### DISCUSSION TITLE
Ab Initio Calculations with PWSCF

### DISCUSSION THREAD

**Discussion by npiroozan** [deepmodeling/dpgen#1168] [Q&A]
Hello,

My apologies but how can we use dpgen with pwscf?  There seem to be detailed descriptions for use with VASP and Abacus, but I don't seen anything for Quantum Espresso.  Any advice in this regard would be very helpful.

Thank you!

**Comment by njzjz** [comment]
Yes but the documentation is missing. You may need to check the source code to see how it works.

https://docs.deepmodeling.com/projects/dpgen/en/latest/run/param.html#run-jdata-fp-style-pwscf

**Reply by npiroozan** [reply]
Great! Thank you!
